build.js

```diff
@@ -1,85 +1,114 @@
+  /* Add Service Worker */
+  await fs.copyFileSync(serviceWorker, `${outDir}/service-worker.js`);
 
-    /* Get an array of every available theme */
-    let themes = await fs.readdirAsync(path.join(assetDir, 'themes'));
+  /* Get an array of every available theme */
+  let themes = await fs.readdirAsync(path.join(assetDir, "themes"));
 
-    if (!themes.includes(theme)) {
-        console.error('Error: Requested theme not found. Defaulting to "light".');
-        theme = 'light';
-    }
-    /* Read in the theme stylesheet */
-    let themeSource = await fs.readFileSync(path.join(assetDir, 'themes', theme));
-    themeSource = themeSource.toString('utf-8');
-    let themeTemplate = hbs.compile(themeSource);
-    let styles = themeTemplate({
-        'background': `${background}`
-    })
-    /* Add the user-specified styles to the new stylesheet */
-    await fs.appendFileAsync(stylesheet, styles);
+  if (!themes.includes(theme)) {
+    console.error('Error: Requested theme not found. Defaulting to "light".');
+    theme = "light";
+  }
+  /* Read in the theme stylesheet */
+  let themeSource = await fs.readFileSync(path.join(assetDir, "themes", theme));
+  themeSource = themeSource.toString("utf-8");
+  let themeTemplate = hbs.compile(themeSource);
+  let styles = themeTemplate({
+    background: `${background}`
+  });
+  /* Add the user-specified styles to the new stylesheet */
+  await fs.appendFileAsync(stylesheet, styles);
 
-    /* Update the config file with the user's theme choice */
-    const data = await getConfig();
-    data[0].theme = theme;
-    await fs.writeFileAsync(config, JSON.stringify(data, null, ' '));
+  /* Update the config file with the user's theme choice */
+  const data = await getConfig();
+  data[0].theme = theme;
+  await fs.writeFileAsync(config, JSON.stringify(data, null, " "));
 }
 
 async function populateConfig(sort, order, includeFork) {
-    const data = await getConfig();
-    data[0].sort = sort;
-    data[0].order = order;
-    data[0].includeFork = includeFork;
-    await fs.writeFileAsync(config, JSON.stringify(data, null, ' '));
+  const data = await getConfig();
+  data[0].sort = sort;
+  data[0].order = order;
+  data[0].includeFork = includeFork;
+  await fs.writeFileAsync(config, JSON.stringify(data, null, " "));
+}
+
+async function populateSocials() {
+  // nothing much to do here as this should be edited by the user
+  fs.copyFile(defaultSocialsPath, social, COPYFILE_EXCL, err => {
+    // file already exists, nothing to do
+  });
 }
 
 async function buildCommand(username, program) {
-    await populateCSS(program);
-    let sort = program.sort ? program.sort : 'created';
-    let order = program.order ? program.order : "asc";
-    let includeFork = program.fork ? true : false;
-    await populateConfig(sort, order, includeFork);
-    updateHTML(('%s', username), sort, order, includeFork);
+  await populateCSS(program);
+  await populateSocials();
+  const socials = await getSocials();

```

Comment:
Could move the `getSocials` calls to updateHTML fn as both `build` and `update` are just passing the values to it.

Reply:
Might as well want to do that with the config, but I left that out for now

Reply:
Done